### DISCUSSION TITLE
How to change scrollbar background color?

### DISCUSSION THREAD

**Discussion by umnyaga** [wezterm/wezterm#6387] [Q&A]
How can I change scrollbar background color.
I found scrollbar_thumb option to change the thumb color, is there similar option for background color?

**Comment by soni17** [comment]
in the` .wezterm.lua` file add the following:


```
config.colors = {
  scrollbar_thumb = 'grey'
}
```

**Reply by umnyaga** [reply]
Nope, that is for the thumb, not for other scroll bar area.
